pkg/agent/controller/egress/egress_controller.go

```diff
@@ -542,6 +554,38 @@ func (c *EgressController) unrealizeEgressIP(egressName, egressIP string) error
 	return nil
 }
 
+func (c *EgressController) uninstallEgressQoSMeter(meterID uint32) error {
+	// Skip the uninstallation on non-Egress Node.
+	if meterID == 0 {
+		return nil
+	}
+	c.meterTrackerMutex.Lock()
+	defer c.meterTrackerMutex.Unlock()
+	if _, exist := c.meterTracker[meterID]; exist {
+		if err := c.ofClient.UninstallEgressQoSMeter(meterID); err != nil {
+			return err
+		}
+		delete(c.meterTracker, meterID)
+	}
+	return nil
+}
+
+func (c *EgressController) installEgressQoSMeter(meterID uint32, rateLimit *crdv1b1.RateLimit) error {
+	if rateLimit == nil {
+		return nil
+	}
+	c.meterTrackerMutex.Lock()
+	defer c.meterTrackerMutex.Unlock()
+	meter, exist := c.meterTracker[meterID]
+	if !exist || meter.Peak != rateLimit.Peak || meter.Burst != rateLimit.Burst {
```

Comment:
No related with the codes, do you know how long it will take affect if there is a new meter configured for an Egress?

Reply:
It should take effect right after the meter and flow are updated. As for the cost of installing a meter, according to the OVS team, it's quite fast, faster than flows.